src/Analyzers/Core/Analyzers/PopulateSwitch/AbstractPopulateSwitchDiagnosticAnalyzer.cs

```diff
@@ -53,7 +53,7 @@ private void AnalyzeOperation(OperationAnalysisContext context)
                 !tree.OverlapsHiddenPosition(switchBlock.Span, context.CancellationToken))
             {
                 Debug.Assert(missingCases || missingDefaultCase);
-                var properties = ImmutableDictionary<string, string>.Empty
+                var properties = ImmutableDictionary<string, string?>.Empty
```

Comment:
hrmm... who passes null for the value.  I'd prefer we attempt to disallow that.  If you make this non-null, can you point to any violators?

Reply:
I don't think we can restrict these the values for the property bag to be non-null. I think I have seen some third party analyzers using null as a value for the property bag entry.